bitnami/redis/templates/metrics-sentinel-svc.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.sentinel.metrics.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis.fullname" . }}-sentinel-metrics
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ template "redis.name" . }}
+    chart: {{ template "redis.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/component: "metrics"
+    {{- if .Values.sentinel.metrics.service.labels -}}
+    {{- toYaml .Values.sentinel.metrics.service.labels | nindent 4 }}
+    {{- end -}}
+  {{- if .Values.sentinel.metrics.service.annotations }}
+  annotations: {{- toYaml .Values.sentinel.metrics.service.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.sentinel.metrics.service.type }}
+  {{ if eq .Values.sentinel.metrics.service.type "LoadBalancer" }}
+  externalTrafficPolicy: {{ .Values.sentinel.metrics.service.externalTrafficPolicy }}
+  {{- end }}
+  {{ if and (eq .Values.sentinel.metrics.service.type "LoadBalancer") .Values.sentinel.metrics.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.sentinel.metrics.service.loadBalancerIP }}
+  {{- end }}
+  ports:
+    - name: sentinelmetrics
+      port: 9355
```

Comment:
This should be based on a parameter (e.g. `sentinel.metrics.service.port`)

Reply:
I'm not sure about this, because this would be inconsistent to the Redis exporter setup, where the port isn't adjustable too.

Reply:
Then, we should fix the redis exporter setup.. But servicer ports should be configurable.

Reply:
I added the variable `sentinel.metrics.service.port` with default value 9355.